### ISSUE TITLE
Support for iCalendar files and CalDAV

### ISSUE THREAD

**Issue by zefr0x** [done-devs/done#44]
**Is your feature request related to a problem? 😢 Please describe.** 
`done` doesn't have any standard method to save and open todo data. To sync them between my different devices (where `done` might not be usable) and multiple different management tools, we need one.

**Describe the solution you'd like**   🤔
Implementing the standard [`iCalendar`](https://www.rfc-editor.org/rfc/rfc5545) files support, maybe optionally by enabling an option and specifying a directory to read from and write to, or specifying a [`CalDAV`](https://en.wikipedia.org/wiki/CalDAV) server to read from and write to.
Most other todo tools and services support `iCalendar` and also `CalDAV`.

`CalDAV` is the main syncing method for a lot of task managing tools including: Nextcloud Tasks, Apple's ICloud and Google Calender ...

This will almost support all task managing cloud services as wanted in the [README.md To do list](https://github.com/done-devs/done/tree/ccfd968b1d99224d645d2772d24204a8b5c7ca51#accounts).

**Additional context** 📝
[`icalendar`](https://crates.io/crates/icalendar) rust library might help implementing this.
Related issues: #11.

**Comment by edfloreshz** [comment]
Thanks for your interest in Done!

I'll add this to the backlog 👍🏼 hopefully I can work on this soon.

**Comment by edfloreshz** [comment]
We have decided to stop developing Done in favor of [Rememberance](https://github.com/dgsasha/remembrance), please consider switching to it.

**Comment by edfloreshz** [comment]
Yes, I was having trouble (and still am) maintaining this app but many people have asked me to continue, so I will.

I'll reopen a bunch of issues I previously closed.
